src/include/btree_inline.h

```diff
@@ -2024,50 +2024,41 @@ __wt_bt_col_var_cursor_walk_txn_read(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *
  *     Return if the cursor is pointing to a page with deleted records and can be skipped for cursor
  *     traversal.
  */
-static inline bool
-__wt_btcur_skip_page(WT_CURSOR_BTREE *cbt)
+static inline int
+__wt_btcur_skip_page(WT_SESSION_IMPL *session, WT_REF *ref, void *context, bool *skipp)
 {
     WT_ADDR_COPY addr;
-    WT_PAGE *page;
-    WT_REF *ref;
-    WT_SESSION_IMPL *session;
     uint8_t previous_state;
-    bool can_skip;
 
-    session = CUR2S(cbt);
-    ref = cbt->ref;
-    page = cbt->ref == NULL ? NULL : cbt->ref->page;
+    *skipp = false; /* Default to reading */
```

Comment:
Styling: comments should be on a separate line.

Reply:
I picked this up from elsewhere in the code. We quite often using same line comments for variable initialization